Gemfile

```diff
@@ -73,3 +72,6 @@ gem 'resque'
 gem 'racecar'
 gem 'resque-multi-job-forks', '~> 0.5.0'
 gem 'resque-scheduler', :git => 'https://github.com/resque/resque-scheduler.git', :ref => 'bbf4930'
+
+gem 'committee'
```

Comment:
This depends on `gem 'openapi_parser', '>= 0.2.2'`, however latest version of that dependency (`0.2.6`, released today) is broken. Until it[1] is fixed, we should stay on `0.2.5`.

On my machine `0.2.6` fails with 
```
bundle exec rake test                          
rake aborted!                                                                                                                                                                                                                              
Bundler::GemRequireError: There was an error while trying to load the gem 'committee'.                          
Gem Load Error is: uninitialized constant OpenAPIParser::DefinitionValidatable                               
Backtrace for gem load error is:                                                                                                                                                                                                           
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `block in load_missing_constant'                                                                
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/active_support.rb:8:in `without_bootsnap_cache'                                                                         
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `rescue in load_missing_constant'
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/active_support.rb:58:in `load_missing_constant'                                                                         
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/openapi_parser-0.2.6/lib/openapi_parser/schemas/openapi.rb:8:in `<class:OpenAPI>'
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/openapi_parser-0.2.6/lib/openapi_parser/schemas/openapi.rb:7:in `<module:Schemas>'                                                                                           
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/openapi_parser-0.2.6/lib/openapi_parser/schemas/openapi.rb:6:in `<top (required)>'                                                                                           
/home/aruzicka/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
```

[1] - https://github.com/ota42y/openapi_parser/issues/34

Reply:
Apparently `0.2.7` was released which should fix this